server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileReset.java

```diff
@@ -79,7 +83,7 @@ public void resetLanguage(String language) {
       ListMultimap<QProfileName, RulesProfile> profilesByName = loadDefinitionsGroupedByName(language);
       for (Map.Entry<QProfileName, Collection<RulesProfile>> entry : profilesByName.asMap().entrySet()) {
         QProfileName profileName = entry.getKey();
-        QualityProfileDto profile = factory.getOrCreate(dbSession, profileName);
+        QualityProfileDto profile = factory.getOrCreate(dbSession, qProfileWsSupport.getDefaultOrganization(dbSession), profileName);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "getDefaultOrganization"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)